thrift/src/test/java/com/linecorp/armeria/common/thrift/text/StackWalkingCurrentThriftMessageClassFinderTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.thrift.text;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.function.Supplier;
+
+import org.junit.jupiter.api.Test;
+
+class StackWalkingCurrentThriftMessageClassFinderTest {
+    Supplier<Class<?>> currentThriftMessageClassFinder;
+
+    {
+        try {
+            currentThriftMessageClassFinder = new StackWalkingCurrentThriftMessageClassFinder();
+        } catch (Throwable throwable) {
+            currentThriftMessageClassFinder = new DefaultCurrentThriftMessageClassFinder();
```

Comment:
You could use an assumption to skip the test on Java 8:

    assumeTrue(SystemInfo.javaVersion() >= 9);